content/blog/2020-05-08-dvc-ambassador-program.md.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Join the DVC Ambassador Program!'
+date: 2020-05-08
+description: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+descriptionLong: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+picture: 2020-05-08/Ambassador_Header.png
+author: marcel_rd
+commentsUrl: https://discuss.dvc.org/t/new-video-reimagining-devops-for-machine-learning/341
+tags:
+  - Ambassador
+  - Volunteer
+  - Talks
+  - Meetup
+  - Outreach
```

Comment:
`Outreach` sound a bit like marketing?

Talks -> Talk? (do we have other tags already similar to this?)

Reply:
we don't have any tags like that. it's kind of a weird tag, doesn't make sense without context.  i changed to 
```
  - Ambassador
  - Volunteer
  - Meetup
  - Blogging